static/app/components/onboarding/gettingStartedDoc/utils/metricsOnboarding.tsx

```diff
@@ -667,3 +667,75 @@ export const getDotnetMetricsOnboarding = ({
     },
   ],
 });
+
+const getUnityConfigureSnippet = () => `
+public override void Configure(SentryUnityOptions options)
+{
+    options.ExperimentalMetrics = new ExperimentalMetricsOptions
+    {
+      EnableCodeLocations = true
+    };
+}`;
+export const getUnityMetricsOnboarding = (): OnboardingConfig => ({
```

Comment:
Nit: as this is used in only one platform I would hardcode it directly in `unity.tsx` 🤷

Reply:
would be nice to apply more abstractions to all the platforms, but let's ,merge for now to unblock the users, and follow up on that later

Reply:
More abstraction potentially makes it harder to change the content and also makes us less flexible. Would be very careful with adding it here. We have to edit the docs content quite often. 🤔
But lets follow up on that 👍